docs/apis/avalanchego/avalanchego-release-notes.md

```diff
@@ -6,6 +6,149 @@
 
 :::
 
+## V1.9.12 [View on GitHub](https://github.com/ava-labs/avalanchego/releases/tag/v1.9.12)
+
+**Banff.12 - Cortina Preparations**
+
+This version is backwards compatible to
+[v1.9.0](https://github.com/ava-labs/avalanchego/releases/tag/v1.9.0).
+It is optional, but encouraged. The supported plugin version is `24`.
+
+### Networking
```

Comment:
I think rather than `### XXX` titles we use `**XXX**` in the rest of this file

Reply:
Yes, otherwise, they become separate titles.

Reply:
Confirmed and fixed.